### ISSUE TITLE
x/tools: TestGuru/describe failing on unsafe.{Add,Slice}

### ISSUE THREAD

**Issue by bcmills** [golang/go#45778]
`TestGuru/describe` seems to be assuming that the `unsafe` package doesn't acquire new identifiers, which causes it to fail on the new identifiers added for #19367 and #40481.

```
--- FAIL: TestGuru (0.00s)
    --- FAIL: TestGuru/describe (1.93s)
        guru_test.go:307: Guru tests for testdata/src/describe/main.go failed: exit status 1.
            --- testdata/src/describe/main.golden	2021-04-22 17:05:18.000000000 +0000
            +++ /workdir/tmp/main.got3261975656	2021-04-23 01:17:04.938640486 +0000
            @@ -22,10 +22,12 @@
             
             -------- @describe unsafe --------
             import of package "unsafe"
            +	builtin Add     
             	builtin Alignof 
             	builtin Offsetof
             	type  Pointer  unsafe.Pointer
             	builtin Sizeof  
            +	builtin Slice   
             
             -------- @describe type-ref-builtin --------
             reference to built-in type float64
            
FAIL
FAIL	golang.org/x/tools/cmd/guru	82.201s
```

[2021-04-26T15:43:18-7c93484/linux-386-longtest](https://build.golang.org/log/cb4d022d630afbc045c983e017d9f596ca0f466e)
[2021-04-26T15:43:18-7c93484/linux-amd64-longtest](https://build.golang.org/log/e6b03d643c251debf4eb1d838c5638c89c547b88)
[2021-04-26T15:26:58-716a04c/linux-386-longtest](https://build.golang.org/log/dddb8370108d46a3d62af2b601f20581fa298b41)
[2021-04-26T15:26:58-716a04c/linux-amd64-longtest](https://build.golang.org/log/88996e227055446b83bf894ca3ce37071bd083ba)
[2021-04-23T21:15:18-f74a669/linux-386-longtest](https://build.golang.org/log/0d032fd69615d23dfb318f2ffc8967d0cf9ddbc8)
[2021-04-23T21:15:18-f74a669/linux-amd64-longtest](https://build.golang.org/log/771e828eccfd57480d131ded2cc347cd3d083c92)
[2021-04-23T19:11:18-e435455/linux-386-longtest](https://build.golang.org/log/10bd8e1c27e60a05b0b26bfc7a3e5e0a790118ed)
[2021-04-23T19:11:18-e435455/linux-amd64-longtest](https://build.golang.org/log/211111eecec57badc0efda978627bb1937dcfc36)
[2021-04-23T17:35:25-f7e8e24/linux-386-longtest](https://build.golang.org/log/cdeba048b2be63a3013284d2961427c8c4162c50)
[2021-04-23T17:35:25-f7e8e24/linux-amd64-longtest](https://build.golang.org/log/e05ec5346d1dce4f96ac4fae54d825a9c3af7c8c)
[2021-04-23T16:19:37-a8e7c0c/linux-386-longtest](https://build.golang.org/log/b0689a47125a59ce481d4938d24cd06a6dac8be6)
[2021-04-23T16:19:37-a8e7c0c/linux-amd64-longtest](https://build.golang.org/log/ea565b936c94cee6fae56aa4f48ca30437e1b925)
[2021-04-22T17:05:18-f946a15/linux-386-longtest](https://build.golang.org/log/6d7d16e7addc430037e43f12d5f10209c79cf2cd)
[2021-04-22T17:05:18-f946a15/linux-amd64-longtest](https://build.golang.org/log/084e3b2acf1d06fc8151a76cdc3577992af7b621)

See previously #44596. Per [my comment](https://go-review.googlesource.com/c/tools/+/296289/5#message-09c953880692c1a1978e7bba3c58da1109409d17) on the fix for that issue, I still wonder if we should just delete that case from `TestGuru`.

CC @mdempsky @findleyr @griesemer

**Comment by bcmills** [comment]
Marking as release-blocker for 1.17 via #11811.

**Comment by findleyr** [comment]
Thanks, I have https://golang.org/cl/313251 out to fix.

**Comment by findleyr** [comment]
Hmm, longtest builders are still failing due to changing position information. I'm sure I tested my fix locally, but perhaps I changed a line number afterward... hmm.

In any case, fix #2 incoming.
